test/dlc_tests/sanity/test_pt_s3.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from invoke.context import Context
+from packaging.version import Version
+
+from test.test_utils import (
+    get_container_name,
+    get_framework_and_version_from_tag,
+    get_processor_from_image_uri,
+    run_cmd_on_container,
+    start_container,
+)
+
+
+@pytest.mark.integration("pt_s3_plugin_sanity")
+@pytest.mark.model("N/A")
+def test_pt_s3_sanity(pytorch_training):
+    """
+    Check that the internally built PT S3 binary is properly installed.
+    :param pytorch_training: framework fixture for pytorch training
+    """
+    _, framework_version = get_framework_and_version_from_tag(pytorch_training)
+    if Version(framework_version) <= Version("1.5.1") and get_processor_from_image_uri(pytorch_training) == "gpu":
+        pytest.skip("Skipping this test for PT 1.5.1 GPU Training DLC images")
```

Comment:
This test should be skipped on all DLC images which are PT 1.5.1 or below, not just GPU images.

You can use the fixture `pt16_and_above_only`, which already exists in `conftest.py`.

Example: https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/ec2/test_smclarify.py#L32

Reply:
Added skip logic to PT version < 1.6 for all tests